projects/fal/src/faldbt/logger.py

```diff
@@ -17,6 +17,17 @@ def __init__(self):
         self._logger = logging.Logger("fal", logging.INFO)
         self._logger.addHandler(self._stdout_handler)
 
+    def __getstate__(self):
+        # Don't pickle the logger
+        d = self.__dict__.copy()
+        d['_logger'] = d['_logger'].name
+        return d
+
+    def __setstate__(self, d):
+        # Set logger when unpickling
+        d['_logger'] = logging.Logger("fal", logging.INFO)
```

Comment:
If you are saving the `.name` of the logger while pickling, would it make sense to do
```suggestion
        d['_logger'] = logging.Logger(d['_logger'], logging.INFO)
```
instead?

Reply:
Maybe

```py
d['_logger'] = logging.Logger(d['_logger'] if isinstance(d['_logger'], str) else "fal", logging.INFO)
```